app/build.gradle

```diff
@@ -53,15 +53,15 @@ android {
 }
 
 ext {
-    supportLibVersion = "26.0.1"
-    espressoVersion = "2.2.2"
-    androidTestVersion = "0.5"
+    supportLibVersion = "27.0.2"
```

Comment:
some of these changed to single quote groovy style (i.e. buildToolsVersion above), could you do it to these too?